table.go

```diff
@@ -101,6 +101,12 @@ func (cc *Conn) FlushTable(t *Table) {
 
 // ListTables returns currently configured tables in the kernel
 func (cc *Conn) ListTables() ([]*Table, error) {
+	return cc.ListTablesOfFamily(TableFamilyUnspecified)
+}
+
+// ListTables returns currently configured tables for the specified table family
+// in the kernel. It lists all tables if family is TableFamilyUnspecified.
+func (cc *Conn) ListTablesOfFamily(family TableFamily) ([]*Table, error) {
```

Comment:
You’re passing `family`, but you’re not using it yet in line 121 (which still reads TableFamilyUnspecified).

Reply:
D'oh! Fixed.